src/pal/inc/pal_mstypes.h

```diff
@@ -218,7 +218,11 @@ typedef __int32 int32_t;
 typedef unsigned __int32 uint32_t;
 typedef __int16 int16_t;
 typedef unsigned __int16 uint16_t;
+
+#if !defined(__ANDROID__)
+// Android already defines these types
 typedef __int8 int8_t;
```

Comment:
It is kind of strange that int8_t is defined on android, but none of the others (int32_t etc) are. Do you know where the typedef comes from on Android?